apps/condo/domains/resident/tasks/helpers/sendBillingReceiptNotifications.js

```diff
@@ -20,8 +24,21 @@ const sendBillingReceiptNotifications = async (context = null) => {
         return 'disabled'
     }
 
-    await sendBillingReceiptsAddedNotifications()
     await sendResidentsNoAccountNotifications()
+
+    const redisClient = await getRedisClient()
+    const redisKey = await redisClient.get(LAST_SEND_BILLING_RECEIPT_NOTIFICATION_DATE)
+
+    if (!redisKey) {
+        await redisClient.set(LAST_SEND_BILLING_RECEIPT_NOTIFICATION_DATE, dayjs().startOf('day').toISOString())
+
+        return 'noRedisKey'
+    } else if (dayjs(redisKey).isSame(dayjs().startOf('day'))) {
+        return
```

Comment:
return type is hard-codeed and inconsistent. This branch return undefined, other - some hard-coded string....

Reply:
```
const SOME_STATUS = 'ok'
const OTHER_STATUS = 'other'
// ...
return { status: ... }
```